src/lua/digest.lua

```diff
@@ -267,4 +313,67 @@ m['aes256cbc'] = {
     end
 }
 
+for _, var in ipairs({'32', '64'}) do
+    local xxHash
+
+    local update_fn_name = 'XXH' .. var .. '_update'
+    local digest_fn_name = 'XXH' .. var .. '_digest'
+    local reset_fn_name = 'XXH' .. var .. '_reset'
+    local copy_fn_name = 'XXH' .. var .. '_copyState'
+    local methods = {
+        update = function(self, str)
+            if type(str) ~= 'string' then
+                error("Usage xxhash" .. var .. ":update(string)", 2)
+            end
+            return builtin[update_fn_name](self.value, str, #str) == builtin.XXH_OK
+        end,
+
+        result = function(self)
+            return builtin[digest_fn_name](self.value)
+        end,
+
+        clear = function(self, seed)
+            if seed == nil then
+                seed = self.default_seed
+            end
+            return builtin[reset_fn_name](self.value, seed) == builtin.XXH_OK
+        end,
+
+        copy = function(self)
+            local copy = xxHash.new()
+            builtin[copy_fn_name](copy.value, self.value)
+            return copy
+        end,
+    }
+
+    local XXH_state_t = ffi.typeof('XXH' .. var .. '_state_t')
+
+    xxHash = {
+        default_seed = 0,
```

Comment:
You can delete this member and inline `0` in its single usage place.

Reply:
I deleted "default_seed" field from "xxHash" table.